contents/projectManagement/githubActions.md

```diff
@@ -0,0 +1,155 @@
+<frontmatter>
+  title: Workflow Automation with GitHub Actions
+  header: pagetop.md
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+<div class="website-content">
+
+{{ booktitle | safe }}
+
+# GitHub Actions
+
+📝 [Alfred Yip](https://github.com/alyip98)
+
+🔎 [James Pang](https://github.com/jamessspanggg), [Tan Yuanhong](https://github.com/le0tan), [Tejas Bhuwania](https://github.com/Tejas2805)
```

Comment:
For consistency, don't use emojis and format authors/reviewers similar to other articles.